src/content/app/species-selector/components/find-gene-instructions/FindGeneInstructions.scss

```diff
@@ -0,0 +1,57 @@
+@import 'src/styles/common';
+
+$columnWidth: 230px;
+$columnGap: 60px;
+$rowGap: 60px;
+$leftMargin: 25px;
+
+.instructionsPanel {
+  margin-top: 15px;
+}
+
+.instructionsWrapper {
+  margin-left: $leftMargin;
+  display: grid;
+  grid-template-columns: repeat(4, #{$columnWidth});
```

Comment:
- Why four columns?

![image](https://user-images.githubusercontent.com/6834224/201078328-2867823e-51f6-42f5-8828-7b1045ecbbc8.png)

(this will have implications for the max-width. However, why do you even need the max-width here?

- Maybe just use flex?

Reply:
I am wondering whether we even need media queries after removing them. The layout hasn't been affected even in iPad mini resolution.

![Screenshot 2022-11-10 at 11 51 20](https://user-images.githubusercontent.com/32512909/201083642-519a4f18-dc17-480d-a744-8af5184c4885.png)

Reply:
> I am wondering whether we even need media queries after removing them. 

Yes; it looks good without them.